src/main/java/com/codingnomads/betty/presentation/controller/PresentationController.java

```diff
@@ -0,0 +1,35 @@
+package com.codingnomads.betty.presentation.controller;
+
+import com.codingnomads.betty.logic.models.SentimentResult;
+import com.codingnomads.betty.logic.services.SentimentAnalyserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+
+import java.sql.SQLOutput;
+import java.util.List;
+
+@Controller
+public class PresentationController {
+
+    private final String text = "'Super amazing.’ Maurizio Sarri assesses our chances in the Europa League and the Premier League top-four race... #CHEBUR";
```

Comment:
Hey man. Not sure we need a Console presentation controller at all here so, you could just delete it as long as the service works. You could just use tests to make sure it works. Also, the input of the Sentiment Analyzer should be a POJO and not just a String. We could call the service by a performing an action in the front-end

Reply:
Hey Jordan, we discussed in the standup today that I'll be doing the integration of the different parts of code (coreNLP, twitter API) into the app and creating the front-end. I can address Cristian's comment here as part of that task. Is this ok with you?